great_expectations/analytics/base_event.py

```diff
@@ -66,10 +70,14 @@ def get_allowed_actions(cls):
     def properties(self) -> dict:
         props = {
             "data_context_id": self.data_context_id,
-            "organization_id": self.organization_id,
             "oss_id": self.oss_id,
-            "service": "python-client",
+            "service": "gx-core",
```

Comment:
This won't break anything right? What's motivating this change?

Reply:
The motivation was just to be more specific, since I could see there being python clients (potentially even including our agent). gx-core seemed most analogous to other service names currently in use (whereas `python-client` seemed too close to the `library` field auto-added by posthog (`posthog-python`)